src/scene.ts

```diff
@@ -4625,6 +4621,11 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
         // Post-processes
         this.postProcessManager.dispose();
 
+        // Components
+        for (let component of this._components) {

```

Comment:
Overall the gamepad component was removed from the list and thus when the camera is disposing the gamepad handlers it was forcing the creation of a new gamepad component ;)